v2/concordium/types.proto

```diff
@@ -2422,7 +2538,11 @@ message LeadershipElectionNonce {
 
 // Response type for GetElectionInfo.
 // Contains information related to baker election for a perticular block.
+// Since the election info is recomputed only at payday? (or is it snapshot?) blocks, the `get_election_info` endpoint returns the same value
+// for the whole payday period.
 message ElectionInfo {
+  // Since the election info is recomputed only at payday? (or is it snapshot?) blocks, the `get_election_info` endpoint returns the same value
+  // for the whole payday period.
```

Comment:
I don't think this should be duplicated. I think the correct thing to say is something like: The `ElectionInfo` will be the same for all blocks in the same epoch. Moreover, from protocol version 4, it is the same for all blocks in the same payday.